src/views/conference/2018/schedule/schedule.jsx

```diff
@@ -0,0 +1,185 @@
+const bindAll = require('lodash.bindall');
+const classNames = require('classnames');
+const connect = require('react-redux').connect;
+const PropTypes = require('prop-types');
+const React = require('react');
+
+const scheduleActions = require('../../../../redux/conference-schedule.js');
+
+const FlexRow = require('../../../../components/flex-row/flex-row.jsx');
+const SubNavigation = require('../../../../components/subnavigation/subnavigation.jsx');
+const TitleBanner = require('../../../../components/title-banner/title-banner.jsx');
+
+const Page = require('../../../../components/page/conference/2018/page.jsx');
```

Comment:
The include of `Page` needs to be after `render`. Page overrides the padding of `#view` to avoid a space between the navigation and the header. If `render` is second the original padding is maintained.